packages/reakit/src/Menu/Menu.tsx

```diff
@@ -0,0 +1,18 @@
+import { theme } from "styled-tools";
+import styled from "../styled";
+import as from "../as";
+import Box, { BoxProps } from "../Box";
+
+const Menu = styled(Box)<BoxProps>`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+
+  &[aria-orientation="horizontal"] {
+    flex-direction: row;
+  }
+
+  ${theme("Menu")};
+`;
+
```

Comment:
I think it's better to accept a `vertical` prop and set `aria-orientation` accordingly. Thus we keep it consistent with other components like [`Toolbar`](https://github.com/reakit/reakit/blob/master/packages/reakit/src/Toolbar/Toolbar.tsx).

Also, it needs a `role` in `defaultProps`:
```suggestion

Menu.defaultProps = {
  role: "menu"
};
```

Reply:
Sounds cool, I guess you meant to name the prop `horizontal` rather than `vertical` ( _right now we're rendering the menu vertically and through a prop we make it horizontal_ ) unless you prefer to do the inverse ?

Reply:
You're right. `horizontal` is better.